core/src/main/resources/META-INF/mule-module.properties

```diff
@@ -144,9 +140,10 @@ artifact.privileged.classPackages=org.mule.runtime.core.internal.interception,\
                                   org.mule.runtime.core.privileged.routing,\
                                   org.mule.runtime.core.privileged.routing.outbound,\
                                   org.mule.runtime.core.privileged.routing.requestreply,\
+                                  org.mule.runtime.core.privileged.transformer,\
                                   org.mule.runtime.core.privileged.transformer.simple
 
-artifact.privileged.artifactIds=com.mulesoft.mule.modules:mule-compatibility-module,com.mulesoft.munit:munit-runner,com.mulesoft.munit:munit-tools
+artifact.privileged.artifactIds=com.mulesoft.mule.modules:mule-compatibility-module,com.mulesoft.munit:munit-runner,com.mulesoft.munit:munit-tools,org.mule.modules:mule-scripting-module
```

Comment:
Should we add an issue to migrate `script:transformer` to `script:component` and add a TODO to remove scripting from privileged api?